launch/display_rover.launch

```diff
@@ -22,6 +23,27 @@
    <node name="mrover_sim_spawn" pkg="gazebo_ros" type="spawn_model" output="screen" args="  -z 0.0 
                         -urdf -param robot_description
                         -model rover" />
+  <node pkg="mapviz" type="mapviz" name="mapviz"></node>
+
+  <node pkg="swri_transform_util" type="initialize_origin.py" name="initialize_origin" >
```

Comment:
what is this for?

Reply:
I actually don't think this is my addition, this was in the code merged in from the `new_urdf` branch

Reply:
you are right, thats my bad. However, new_urdf made it into master so I'm surprised that git shows this as part of the diff. Weird. Even weirder is that this isn't listed in merge conflicts, so this might do something weird when u go to merge. Just make sure it doesn't duplicate.